packages/lib-classifier/src/store/feedback/helpers/get-feedback-from-tasks.js

```diff
@@ -1,17 +1,18 @@
-import _ from 'lodash'
-
 // Filters a workflow's tasks object to an object of `taskId: [feedback rules]` pairs
-function getFeedbackFromTasks (tasks) {
-  return _.reduce(tasks, (result, task, taskId) => {
-    if (_.get(task, 'feedback.enabled', false) &&
-      _.get(task, 'feedback.rules', []).length > 0) {
-      return _.assign({}, result, {
-        [taskId]: task.feedback.rules
-      })
-    } else {
-      return result
+function getFeedbackFromTasks (tasks = {}) {
+  const result = {}
+  Object.entries(tasks).forEach(function getTaskRules ([taskKey, task]) {
+    const { feedback } = task
+    const taskHasFeedback =
+      feedback &&
+      feedback.enabled &&
+      feedback.rules.length > 0
```

Comment:
Maybe `feedback.rules && feedback.rules.length > 0`?

Reply:
I've updated to add a check for `rules` and a default value of `{ enabled: false, rules: [] }` for the feedback object.